server/controllers/posts.js

```diff
@@ -111,10 +112,14 @@ export const getPostsBySearch = async (req, res) => {
 					$or: [{ creator: userId }, { private: false }],
 				},
 				{
-					$or: [{ title }, { tags: { $in: tags.split(',') } }],
+					$and: [
```

Comment:
Explain the changes u made in this query

Reply:
I have made this changes beacuse your tags search was giving all posts on searching specific tags  and empty title ,with and it is working fine